src/components/SettingsMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import { IoClose } from "solid-icons/io";
+
+import { useGlobalContext } from "../context/Global";
+import "../style/settings.scss";
+import Denominaton from "./Denomination";
+import Separator from "./Separator";
+import Tooltip from "./Tooltip";
+
+const SettingsMenu = () => {
+    const { t, settingsMenu, setSettingsMenu } = useGlobalContext();
+
+    return (
+        <div
+            id="settings-menu"
+            class="frame assets-select"
+            onClick={() => setSettingsMenu(false)}
```

Comment:
Not a fan of this. I would rather expect it to close on an outside click

Reply:
basically the same from assetselect, the pop over has an :after element fullsize which darkens everython in the background, if click anywaere else but the relevant inputs the popover will close

Reply:
> if clicked anywhere else but the relevant inputs the popover will close

which feels weird to me, but I wont die on this hill

Reply:
I agree it's weird and it's only "working" on asset select because the actionable buttons are that large that it's hard to misclick. I agree it should only close on [x] and clicking outside, let's change behavior for both.

Reply:
> I agree it's weird and it's only "working" on asset select because the actionable buttons are that large that it's hard to misclick. I agree it should only close on [x] and clicking outside, let's change behavior for both.

Can be done in a separate PR if you prefer